### ISSUE TITLE
Need for more features and filters 

### ISSUE THREAD

**Issue by NitikaBahl** [NitikaBahl/Python-Projects#1]
I have added a simple GUI and am looking to enhance its features and add more filters.

**Comment by red-threat** [comment]
programming language?

**Comment by NitikaBahl** [comment]
The code is in Python.

**Comment by red-threat** [comment]
Filtering the data or enhancing the GUI?

**Comment by NitikaBahl** [comment]
In image processing applications with a graphical user interface (GUI), adding more filters typically requires updating the GUI to allow users to select and apply these filters easily. Also, I would appreciate it if you could help me make the GUI better.

**Comment by red-threat** [comment]
Yes, I can help you in upgrading your GUI to add new filters that can further enhance the user experience.

**Comment by NitikaBahl** [comment]
ok perfect! I'll then be sure to assign you the issue.
